app/(tabs)/(index,host)/event/[id]/index.tsx

```diff
@@ -79,6 +83,16 @@ export default function EventPage() {
     });
   }
 
+  const submitCancelAttendance = async () => {
+    await cancelAttendance.mutateAsync();
+    showToastable({
+      message: "Attendance cancelled",
+      duration: 2000,
+      status: "success",
+    });
+    router.push("/(tabs)/(host)");
```

Comment:
This should probably be either `router.back()` or just left out, leaving you on the event page